VideoLocker/src/main/java/org/edx/mobile/model/course/BlockType.java

```diff
@@ -4,6 +4,12 @@
  * Created by hanning on 5/21/15.
  */
 public enum  BlockType {
-    COURSE, CHAPTER,  SECTION,  SEQUENTIAL,
-    VERTICAL,  VIDEO,  HTML, PROBLEM, DISCUSSION, OTHERS
+    COURSE(false), CHAPTER(false),  SECTION(false),  SEQUENTIAL(false),
```

Comment:
Hrrrm. I'm always in favor of ensuring the compiler checks things. I hadn't realized Java doesn't properly deal with enum exhaustivity - but I guess it's necessary since ``null`` is a valid input.

*@aleffert reads section in Effective Java*

I guess if we do add more fields, we can factor this to be methods on an abstract type or something like that, so we don't just end up with a bunch of constructors with unnamed properties that are hard to read at a glance.